### ISSUE TITLE
Validator returns "Congratulations! No Error Found." for CSS with syntax errors in it

### ISSUE THREAD

**Issue by kupietools** [w3c/css-validator#429]
The following CSS validates correctly:

```
b {text-decoration: underline solid black;}
/* .disabled-attribute{display: block;} */   %
!
)
/* .another > .disabled > .attribute {box-shadow: 10px 10px 20px RGBA(255,255,255,0);border: 1px solid hsla(0,0%,100%,.2) !important;} */

span.greentext {color: green;} /* dummy because theme bug is removing first A rule for some reason */
```

Instead of reporting the error when it encounters the invalid characters, it simply stops validating and reports that the CSS is valid.

I've put together a working example of this at [https://github.com/kupietools/W3C-CSS-validation-bug](https://github.com/kupietools/W3C-CSS-validation-bug).
